ciment/gildas/default.nix

```diff
@@ -21,7 +21,6 @@ stdenv.mkDerivation rec {
   configurePhase=''
     substituteInPlace admin/wrapper.sh --replace '%%OUT%%' $out
     source admin/gildas-env.sh -b gcc -c gfortran -o openmp
-    export GAG_INC_FLAGS=""
```

Comment:
@bzizou This envar is needed to pass the include path along. Do you remember why it was unset in the first place?

Reply:
No, I don't remember...